spring-data-sample/README.md

```diff
@@ -129,7 +129,7 @@ Please note that starting the containers may take more than one minute.
 
 ### Initial data
 
-After the containers have loaded, you need to load the initial data by running the following command:
+After the containers have started, you need to load the initial data by running the following command:
```

Comment:
We've already used term `load` for sample records in the tables on this tutorial. Also `docker` command has `load` sub-command for Docker container images. So, I think other term would avoid confusions.